src/helpers.js

```diff
@@ -155,3 +155,99 @@ export const isRectSelected = rect => selection => {
 
   return true;
 };
+
+export const transpose = m => m[0].map((x, i) => m.map(x => x[i]));
```

Comment:
what is `m`? do you think it's readable enough for people without context? Could we make `m` a bit more meaningful and add a comment how `transpose` is meant to be used?

Reply:
My bad, I thought transpose would be a common concept but it's not, let me improve this function.